ui/e2e/src/config/base.playwright.config.ts

```diff
@@ -69,6 +64,10 @@ const config: PlaywrightTestConfig = {
       name: 'chromium',
       use: {
         ...devices['Desktop Chrome'],
```

Comment:
Caught this while working on another PR. This spread was overriding the global defaults, which wasn't obvious -- there were a few github issues on playwright where people hit the same issue.

For now, inlining the correct value in the chrome config. If we ever expand to multiple browsers, we'll likely set it as a constant that's used across them.